assets/src/scss/layout/_cookies-new.scss

```diff
@@ -0,0 +1,58 @@
+.cookie-notice.new-design {
```

Comment:
If this markup is very similar to the old version, and we're pretty sure we'll move to this new version, we could do the same thing as with beta blocks. Copy the old version to a new file to freeze it, and make the modifications to the existing file, so that it's easier to track history.

Reply:
Not sure how that would work with the A/B test in Optimize? 🤔

Reply:
It could still contain the same CSS. I mean, if we freeze the old version by copying the file and adding `-old` to the file name, then the PR can modify the existing file name and it would be easy to see which changes are made in the new version.

Reply:
Just a suggestion, could be in this case the history is not that useful :shrug: